tests/core/models.py

```diff
@@ -92,20 +113,24 @@ class MyDefaultPKModel(models.Model):
     id = models.IntegerField(primary_key=True, default=lambda: next(int_source), editable=False)
     content = models.TextField(blank=True, default='')
 
+    class Meta:
+        app_label = 'core'
+
+
+class MyUUIDModel(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    anotheruuid = models.UUIDField(default=uuid.uuid4)
+    content = models.TextField(blank=True, default='')
+    order = models.IntegerField(default=0, blank=True)
+
+    class Meta:
+        ordering = ('order',)
+        app_label = 'core'
 
-if hasattr(models, 'UUIDField'):
```

Comment:
Every version of Django that we support now has models.UUIDField, so this check is no longer needed.